### DISCUSSION TITLE
v2.0 - Search all dimensions/metrics (including "Simple") by default

### DISCUSSION THREAD

**Discussion by jmcapra** [apache/superset#24621] [Ideas]
## Screenshot

<img width="320" alt="image" src="https://github.com/apache/superset/assets/35404590/072c7675-bdc0-45cb-a8e2-2d08dbfa3d78">

## Description

In version 2.0, adding a new "metric" or "dimension" filter in an /explore chart has the following tabs: "Saved", "Simple", and "Custom SQL". The default is "saved". Previously, the default tab allowed you to search both "Saved" and "Simple" together. The previous functionality was preferable - could we add an "All" tab as the default? 90% of the time I want to use a "simple" expression as the metric/dimension, and this requires switching to "Simple" before searching. Just show all results, with the "computed"/"saved" icon to indicate saved metrics/dimensions. 

## Design input
design:suggest

**Comment by rusackas** [comment]
I think it would make sense to at least make the default tab a configuration option, and have the default configuration reflect the current behavior. That would seemingly be unobjectionable, and you could customize it as you like. 

For changing the default behavior/design, I'll ping @kasiazjc for feedback, and link to this Discussion on the `#product-designers` channel on [slack](http://bit.ly/join-superset-slack)
